sympde/topology/basic.py

```diff
@@ -535,18 +535,18 @@ def patches(self):
     @property
     def interfaces(self):
         ls = []
-        data = OrderedDict(sorted(self._data.items()))
+        data = dict(sorted(self._data.items()))
         for _,v in data.items():
             ls.append(v)
         return Union(*ls)
 
     def todict(self):
         # ... create the connectivity
         connectivity = {}
-        data = OrderedDict(sorted(self._data.items()))
+        data = {k: v for k,v in sorted(self._data.items())}
```

Comment:
As you have done in other places, you can just write
```suggestion
        data = dict(sorted(self._data.items()))
```

Reply:
I must have missed it because my last commit was all about putting `dict(...)` instead of `{k: v ....}`